modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc

```diff
@@ -1,14 +1,49 @@
-[id="anchor-and-file-names"]
-= Anchor Names and File Names
+[id="module_anchor-and-file-names-concept"]
+= File Names and Anchors
 
-To optimize modular documentation, follow these guidelines for naming module anchors and files:
+To optimize modular documentation, follow these guidelines for naming files and creating anchors:
 
-Anchor names:: Provide an anchor in the format `+++[id="anchor-name_{context}"]+++` for every module so that it can be identified by Asciidoctor when reused or cross-referenced. `+++{context}+++` is a variable whose value you define in the assembly. Give the anchor the same or similar name as the module heading. Separate the words in the anchor with hyphens and use an underscore to separate the `+++{context}+++` part:
-+
+.File names
+Assembly and module file names should accurately and closely reflect the title of the assembly or module. File names should have the format `prefix-filename.adoc` or `prefix_filename.adoc` where `prefix` is one of the following module prefixes:
+
+* `con`: Concept module prefix
+* `proc`: Procedure module prefix
+* `ref`: Reference module prefix
+* `assembly`: Assembly module prefix
+
+.Examples
+* `con-guided-decision-tables.adoc`  (Concept module)
+* `proc-creating-guided-decision-tables.adoc`  (Procedure module for creating)
+* `proc-editing-guided-decision-tables.adoc`  (Procedure module for editing)
+* `ref-guided-decision-table-examples.adoc`  (Reference module with examples)
+* `ref-guided-decision-table-columns.adoc`  (Reference module with column types)
+* `assembly-designing-guided-decision-tables.adoc`  (Assembly of guided decision table modules)
+
+
+[NOTE]
+====
+Do not include special characters in file names. Ensure that all members of your team use the same file naming conventions.
+====
+
+These file naming guidelines are optional but highly recommended. However, if your team does not include the module prefixes in file names followed by either a hyphen (-) or an underscore (_), you must include one of the following variables in each concept, procedure, and reference module:
+
+NOTE: This is a Pantheon 2 requirement.
```

Comment:
I think maybe nix this note? It's just irrelevant in Pantheon 1, plus all of this is really in preparation for Pantheon 2. If you want to leave it, I suggest just rewording to "This is supported in Pantheon 2 only." instead of calling it a requirement. Makes it seem at a glance that even if you do name your files with the correct prefix, still required to have these variables. IDK. Your call.

Reply:
Again, this goes hand in hand with the above suggestion. This is a hard Pantheon 2 requirement, so we should mention it, but we should present both options as equal.

Reply:
@vikram-redhat Sure, I was saying more to modify so that it's clearer that the prefix or variable is a requirement, one or the other. This not here to me makes it seem like the variable is the hard requirement, regardless of the prefix (meaning, even if you use the prefix, also have to use the variable as a requirement, which isn't what we mean). I suggest we just explain that requirement as part of the instruction in the text, or if we like the note, then put it somewhere earlier so it's clear it applies to both options.

Reply:
@sterobin, I think that we need a separate section to discuss the Pantheon 2 requirements. I think it would be clearer to present the file name guidance as a best practice here in isolation. In the requirements section, we could explain that you must mark the module file types. One way is to use the recommended naming practices. If you can't use the recommended naming practices, then you need to add these attributes.

Reply:
@kalexand-rh Sure, I'll leave it up to @emmurphy1 if she wants to do that or just mention in passing as she has.

Reply:
@vikram-redhat, I think we should not call out "Pantheon" at all in these guidelines. They are available and promoted for use in upstream docs projects and in other organizations. References to internal-only tooling and/or conditioning any of the rules/guidelines on the use of a particular version of internal tooling turns this content into a RH-only thing. I don't think we want that.

Reply:
I agree with @rkratky and @kalexand-rh and combining those two suggestions would make sense. Have a separate Pantheon 2 section which essentially lists the requirements for making these guidelines work with RH tooling.

Reply:
OK, that makes sense. I'll remove the Pantheon 2 note, @vikram-redhat . @kalexand-rh  would you like to create the Pantheon 2 section? However, before you do, about @rkratky's point to  not include anything about Pantheon 2 in this guide.  @rkratky would you feel comfortable if we  add a separate Pantheon section or do you feel strongly that we shouldn't mention Pantheon at all?